fcp/loadplugin.py

```diff
@@ -0,0 +1,100 @@
+"""
+load a freenet plugin
+
+This is the guts of the command-line front-end app fcploadplugin
+"""
+
+#@+others
+#@+node:imports
+import argparse
+import sys
+import traceback
+import os.path
+from urlparse import urlparse
+
+from . import node
+from .arguments import add_default_arguments
+
+#@-node:imports
+#@+node:globals
+
+#@-node:globals
+#@+node:usage
+def create_parser():
+    '''
+    Creates an argparse parser.
+    '''
+    parser = argparse.ArgumentParser(
+        prog='fcploadplugin',
+        description='''
+            A simple command-line freenet load plugin command.
+
+            Loads a plugin to the running node.
+        ''',
+    )
+    add_default_arguments(parser)
+
+    parser.add_argument(
+        'plugin_uri',
+        help='''
+            A URI that points to the plugin location. 
```

Comment:
It might be good to clarify that this can either be a path in the filesystem (looks assumed to be the same as the node's) or a Freenet key.